static/gsApp/types/index.tsx

```diff
@@ -879,13 +926,31 @@ type PendingReservedBudget = {
 };
 
 export type ReservedBudget = {
+  /**
+   * The categories included in the budget and their respective ReservedBudgetMetricHistory
+   */
   categories: Partial<Record<DataCategory, ReservedBudgetMetricHistory>>;
+  /**
+   * The amount of free budget gifted in the associated usage cycle
+   */
   freeBudget: number;
+  /**
+   * The id of the ReservedBudgetHistory object
+   */
   id: string;
+  /**
+   * The percentage of the budget used, including gifted budget
+   */
   percentUsed: number;
+  /**
+   * The amount of budget in the associated usage cycle, excluding gifted budget
+   */
   reservedBudget: number;
+  /**
+   * The amount of budget used in the associated usage cycle
+   */
   totalReservedSpend: number;
-};
+} & ReservedBudgetCategory;
```

Comment:
from this change in the backend https://github.com/getsentry/getsentry/pull/17390/files#diff-25cc2a5d6c3d4492458175fde0902fe97992dca9c600ec1fc9529271da7e4dcb